### ISSUE TITLE
apiFetch in control_new should pull the CSRF token from the user's cookies

### ISSUE THREAD

**Issue by Osmose** [mozilla/normandy#865]
Django stores the CSRF token in the user's cookies; we should be pulling it from there instead of redundantly adding it to the HTML for the page. I'm not entirely sure, but this might also help with stale CSRF tokens, as a failed request to the service can return an updated cookie with a fresh token.

**Comment by rehandalal** [comment]
+1

**Comment by rehandalal** [comment]
Hmm... I tried to do this and couldn't get it to work. It doesn't appear to be setting the CSRF cookie for some reason. I tried changing `CSRF_COOKIE_HTTPONLY` to `False`, as well as `CSRF_COOKIE_SECURE` to `False` (since the cert is not valid in development). But still no dice!

**Comment by Osmose** [comment]
I got your PR working on the extensions page with two changes:

1. Set the default value for the `errors` prop in `BaseAntForm` to an empty object. This is a leftover bug from the eslint changes that will go away when `BaseAntForm.js` is replaced by `forms.js` in #859.
2. Set `CSRF_COOKIE_HTTPONLY` to False. Ideally this should only be on writable admin nodes; read-only ones should not bother setting CSRF stuff at all.

I also had to clear my cookies for this to work, as the existing CSRF cookie was prevent Django from setting a new one with `httponly` set to False.
